orthoinference/src/main/java/org/reactome/orthoinference/OrthologousEntityGenerator.java

```diff
@@ -280,43 +299,46 @@ private static GKInstance createInfEntitySet(GKInstance entitySetInst, boolean o
 			// Filtering based on ProteinCount results
 			if (!override && entitySetTotalCount > 0 && entitySetInferrableCount == 0)
 			{
-				logger.info("\tNo distinct proteins found in EntitySet -- terminating inference");
+				logger.info("No distinct proteins found in EntitySet -- terminating inference");
 				return nullInst;
 			}
 			
 			if (entitySetInst.getSchemClass().isa(CandidateSet))
 			{
-				Set<String> existingCandidateInstances = new HashSet<String>();
-				List<GKInstance> candidatesList = new ArrayList<GKInstance>();
+				Set<String> existingCandidateInstances = new HashSet<>();
+				List<GKInstance> infCandidatesList = new ArrayList<>();
 				// Equivalent to infer_members function in infer_events.pl
-				logger.info("\tInferring CandidateSet candidate instances...");
-				for (GKInstance candidateInst : (Collection<GKInstance>) entitySetInst.getAttributeValuesList(hasCandidate))	
+				Collection<GKInstance> candidateInstances = (Collection<GKInstance>) entitySetInst.getAttributeValuesList(hasCandidate);
+				logger.info("Total candidate instances: " + candidateInstances.size());
+				logger.info("Candidate instances: " + candidateInstances);
+				for (GKInstance candidateInst : candidateInstances)
 				{
 					GKInstance infCandidateInst = createOrthoEntity(candidateInst, false);
 					if (infCandidateInst != null && !existingMemberInstances.contains(infCandidateInst.getAttributeValue(name).toString()) && !existingCandidateInstances.contains(infCandidateInst.getAttributeValue(name).toString()))
 					{
 						existingCandidateInstances.add(infCandidateInst.getAttributeValue(name).toString());
-						candidatesList.add(infCandidateInst);
+						infCandidatesList.add(infCandidateInst);
 					}
 				}
+				logger.info("Total number of inferred candidates: " + infCandidatesList.size() + "/" + candidateInstances.size());
 				// Handling of CandidateSets
-				if (candidatesList.size() > 0)
+				if (infCandidatesList.size() > 0)
```

Comment:
Could the deep level of nesting here be changed for guard clauses or using else-if branches?